classes/class-ledyer-lco-gateway.php

```diff
@@ -399,14 +457,14 @@ public function add_data_attributes( $tag, $handle ) {
 				$lco_order_id     = WC()->session->get( 'lco_wc_session_id' );
 
 				if ( is_order_received_page() ) {
-					$order_key    = filter_input( INPUT_GET, 'key', FILTER_SANITIZE_STRING );
+					$order_key = filter_input( INPUT_GET, 'key', FILTER_SANITIZE_STRING );
 
```

Comment:
`FILTER_SANITIZE_STRING` is deprecated. Change this to be `FILTER_SANITIZE_FULL_SPECIAL_CHARS`